mpmath/matrices/linalg.py

```diff
@@ -592,6 +598,10 @@ def det(ctx, A):
         try:
             # do not overwrite A
             A = ctx.matrix(A).copy()
+
+            if A.rows == 0 or A.cols == 0:
+                return 1
```

Comment:
Can't we fix instead the ``LU_decomp()``, without a special case here?  E.g. for ``p = []`` the code below will return ``1``.

To fix exception in the ``min()`` builtin you could provide appropriate default.

Reply:
Yes, that would probably make for a better solution.
Adding a default value to the builtin in `mnorm()` seems to have fixed it, but still had to add an extra condition to an if caluse in `LU_decom()`.